docs/capability-types.md

```diff
@@ -0,0 +1,130 @@
+This is a full list of capability types with their properties, entities and units. See the [general information about capabilities](fixture-format.md#capabilities).
+
+### Possible entities and keywords
+
+| Entity | Allowed Units | `-100%` | `-1%` | `0%` | `1%` | `100%`
+| - | - | - | - | - | - | -
+| Speed | `Hz`, `%` | fast reverse | slow reverse | stop | slow | fast
+| RotationSpeed | `Hz`, `%` | fast CCW | slow CW | stop | slow CW | fast CWW
+| Time | `s`, `ms`, `%` | – | – | instant | short | long
+| Distance | `m`, `%` | – | – | – | near | far
+| Brightness | `lm`, `%` | – | – | off | dark | bright
+| ColorTemperature | `K`, `%` | – | – | – | warm | cold
+| FogOutput | `m³/min`, `%` | – | – | off | weak | strong
+| RotationAngle | `°`, `%` | – | – | – | – | –
+| BeamAngle | `°`, `%` | – | – | closed | narrow | wide
+| Factor | (no unit), `%` | – | – | off | low | high
+| Index | (no unit) | – | – | – | – | –
+| Percent | `%` | – | – | off | low | high
+| Insertion | `%` | – | – | out | – | in
+| IrisPercent | `%` | – | – | closed | – | open
+
+### Possible capability types
+
+Required properties are _italic_, the entity is written in (parentheses).
+
+* Nothing
+* ShutterOpen
+* ShutterClosed
+* StrobeEffect
+  * strobeType: one of `Normal` (default), `Random`, `Pulse`, `PulseRandom`, `RampUp`, `RampUpRandom`, `RampDown`, `RampDownRandom`
+* StrobeSpeed
+  * _speed_ (Speed)
+  * strobeType: one of `Normal` (default), `Random`, `Pulse`, `PulseRandom`, `RampUp`, `RampUpRandom`, `RampDown`, `RampDownRandom`
+* StrobeDuration
+  * _time_ (Time)
+  * strobeType: one of `Normal` (default), `Random`, `Pulse`, `PulseRandom`, `RampUp`, `RampUpRandom`, `RampDown`, `RampDownRandom`
+* Intensity
+  * _brightness_ (Brightness)
+* ColorIntensity
+  * _color_: one of our predefined Single Colors (Red, Green, Blue, White, Amber, etc.)
+  * _brightness_ (Brightness)
+* Color
+  * colors: array of visible colors as hex codes; e.g. `['#ffff00']` if red and green LEDs are on that mix to yellow, `['#ff0000', '#00ff00']` if red and green laser beams are on that don't mix. UV is always counted as seperate color.
+  * use colorsStart and colorsEnd instead of colors to declare a color change from the start to the end of the range
```

Comment:
So this is a special case? Maybe we should rather find a solution that works for all capability types instead? What about this:

```js
{
  "dmxRange": [0, 20],
  "type": ...,
  // other properties that apply to the whole range
  "start": {
    // properties that apply to DMX value 0
  },
  "end": {
    // properties that apply to DMX value 20
  }
}
```

Reply:
Solution: Instead add start and end variants for all properties. E.g.:

```js
{
  "dmxRange": [0, 127],
  "type": "Speed",
  "speedStart": "slow",
  "speedEnd": "fast"
  // "speed" is forbidden
},
{
  "dmxRange": [128, 255],
  "type": "Speed",
  "speed": "70%",
  // "speedStart" and "speedEnd" are forbidden
}
```

Reply:
Done.